ui/src/VoxettaApp.js

```diff
@@ -30,3 +26,36 @@ export class VoxettaApp extends LitElement {
 }
 
 customElements.define('vox-app', VoxettaApp);
+
+
+navigator.mediaDevices.getUserMedia({ audio: true, video: false })
+    .then(function(stream) {
+
+        const recordButton = document.getElementById("recordButton");
+        const audioSave = document.getElementById("utterance");
```

Comment:
as stated before, decouple the dom manipulation from the audio recording. This audio recording logic should be able function independent of any specific button being present on DOM.